main.js

```diff
@@ -0,0 +1,354 @@
+        combo[1].style.setProperty('color', 'gold');
+        combo[2].style.setProperty('color', 'gold');
+        clearInterval(timeInterval);
+        return true;
+      }
+    }
+  }
+}
+
+// Game Timer --------------------------------------------------------------------------------------------------------------------------------------------
+function gameTimer() {
+  // keeps track of seconds in ones place ----------------------------------------------------------------------------------------------------------------
+  if (secOnes <= 9) {
+    secOnes += 1;
+    timeDiv.textContent = `${hourTens}${hourOnes}:${minTens}${minOnes}:${secTens}${secOnes}`;
+    //reset to 0, add 1 to secTens
+    if (secOnes === 10) {
+      secOnes = 0;
+      secTens += 1;
+    }
+  }
+
+  // keeps track of seconds in tens place ----------------------------------------------------------------------------------------------------------------
+  if (secTens <= 5) {
+    timeDiv.textContent = `${hourTens}${hourOnes}:${minTens}${minOnes}:${secTens}${secOnes}`;
+  }
+  // reset to 0, add 1 to minOnes
+  else if (secTens === 6) {
+    secTens = 0;
+    minOnes += 1;
+  }
+
+  // keeps track of minutes in ones place ----------------------------------------------------------------------------------------------------------------
+  if (minOnes <= 9) {
+    timeDiv.textContent = `${hourTens}${hourOnes}:${minTens}${minOnes}:${secTens}${secOnes}`;
+  }
+  //reset to 0, add 1 to minTens
+  else if (minOnes === 10) {
+    minOnes = 0;
+    minTens += 1;
+  }
+
+  // keeps track of minutes in tens place ----------------------------------------------------------------------------------------------------------------
+  if (minTens <= 5) {
+    timeDiv.textContent = `${hourTens}${hourOnes}:${minTens}${minOnes}:${secTens}${secOnes}`;
+  }
+  // reset to 0, add 1 to hourOnes
+  else if (minTens === 6) {
+    minTens = 0;
+    hourOnes += 1;
+  }
+
+  // keeps track of hours in ones place ------------------------------------------------------------------------------------------------------------------
+  if (hourOnes <= 9) {
+    timeDiv.textContent = `${hourTens}${hourOnes}:${minTens}${minOnes}:${secTens}${secOnes}`;
+  }
+  // reset to 0, add 1 to hourTens
+  else if (hourOnes === 10) {
+    hourOnes = 0;
+    hourTens += 1;
+  }
+
+  // Keeps Track of hours in tens place ------------------------------------------------------------------------------------------------------------------
+  if (hourTens <= 9) {
+    timeDiv.textContent = `${hourTens}${hourOnes}:${minTens}${minOnes}:${secTens}${secOnes}`;
+  }
+  //Stops if it becomes greater than 9
+  else if (hourTens === 10) {
+    timeDiv.textContent = 'I think you have spent way too much time playing a single tic tac toe game.';
+  }
+}
+
+// Computer AI (NOT WORKING) -----------------------------------------------------------------------------------------------------------------------------
+function computerAi() {
```

Comment:
Start with the simple AI, where the computer just picks a random square rather than trying to play the best move.